File: BusinessApp.API/Models/Business.cs
using System.Collections.Generic;

namespace BusinessApp.API.Models
{
    public class Business
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Address { get; set; }
        public string OfficeNumber { get; set; }
        public string Landmark { get; set; }
        public bool IsPublished { get; set; }
        public bool IsPublishable { get; set; } 
        public User User { get; set; }
        public int UserId { get; set; }
        public Video Video { get; set; }
        public ICollection<Photo> Photos { get; set; }
    }
}

File: BusinessApp-SPA/src/app/_models/Business.ts
import { Photo } from './Photo';
import { Video } from './Video';

export interface Business {
    id: number;
    name: string;
    description?: string;
    address?: string;
    officeNumber: string;
    userId: number;
    photoUrl: string;
    validTill: number;
    photos?: Photo[];
    video?: Video;
}
